### ISSUE TITLE
Unit: N/A, NA, etc.

### ISSUE THREAD

**Issue by rmx90210** [CityOfPhiladelphia/passyunk#44]
We received some feedback from a Property Search user who didn't get any results because they entered `NA` for the unit. Should these be stripped out by Passyunk?

* N/A
* NA
* None
* unknown
* etc.

**Comment by tswanson** [comment]
I've been monitoring the 404 returns for AIS and this is the first time I've seen this.  Property needs a single field entry.

**Comment by rmx90210** [comment]
Agree, single field would help this. But I could still see it coming over in voters or something other manually-entered dataset. Just something to think about.
